core/src/test/java/greencity/validator/TipsAndTricksDtoRequestValidatorTest.java

```diff
@@ -24,13 +25,15 @@
     @Mock
     private TagsService tagService;
 
+    private static final TagType TAG_TYPE = TagType.TIPS_AND_TRICKS;
```

Comment:
As I see, you can just use TagType.TIPS_AND_TRICKS as parameter in method. No need to create additional field